### PR TITLE
[SCons] Fix a few more missing imports

### PR DESCRIPTION
**Pull request by bryanwweber**
**Changes proposed in this pull request**

- Fixes some missing imports in the SConscript files

Closes https://groups.google.com/g/cantera-users/c/7TsfXeWGIy8

**If applicable, provide an example illustrating new features this pull request is introducing**

**Checklist**

- [x] The pull request includes a clear description of this code change
- [x] Commit messages have short titles and reference relevant issues
- [x] Build passes (`scons build` & `scons test`) and unit tests address code coverage
- [x] Style & formatting of contributed code follows [contributing guidelines](https://github.com/Cantera/cantera/blob/main/CONTRIBUTING.md)
- [x] The pull request is ready for review

### CODE REVIEW

**Review by ischoegl — COMMENTED** [review]
Thanks for the fix, @bryanwweber! One small comment though: while it is likely a deliberate choice to stick with `os.path` and keep the diff small, I _believe_ the eventual objective is to move to `pathlib`?

**Comment by bryanwweber** [comment]
@ischoegl Yes, that is the eventual objective. I didn't want this quick fix to conflict with those changes later on. I don't know when I'll have time to get back to that project 😞

**Review by ischoegl — DISMISSED** [review]
Changes look good to me (not sure why it worked before).

I will wait with the merge though as I'd like to hear back from the [UG report](https://groups.google.com/g/cantera-users/c/7TsfXeWGIy8).

**Comment by bryanwweber** [comment]
I think it worked in the past because the `relpath` is only called when there's a space in the path, which is most common on Windows in my experience (hence the UG report). I actually already had this branch and just hadn't made a PR for it, I was going to add a few other fixes possibly, but since there was the report I figured I'd better get it into main.

File: samples/f77/SConscript [hunk 1]
```diff
@@ -34,7 +34,7 @@ mak_path = pjoin(localenv['ct_incroot'], 'cantera', 'Cantera.mak')
 if ' ' in mak_path:
     # There is no reasonable way to handle spaces in Makefile 'include'
     # statement, so we fall back to using the relative path instead
-    mak_path = os.path.relpath(mak_path, pjoin(localenv['ct_sampledir'], 'f90'))
+    mak_path = relpath(mak_path, pjoin(localenv['ct_sampledir'], 'f90'))
```
Comment (bryanwweber) [inline samples/f77/SConscript]: I'm wondering if this should be `'f77'` at the end of the line?

Comment (ischoegl) [reply samples/f77/SConscript]: I believe this is correct - it's in the `samples/f77` folder, after all. Please update and I can merge. The math constant issue that popped up in the same UG post is somewhat gnarlier, and very different from what is addressed here.

Comment (bryanwweber) [reply samples/f77/SConscript]: @ischoegl Done! Used the new `github.dev` editor, it was pretty seamless and nice!

**Review by ischoegl — APPROVED** [review]

**Comment by ischoegl** [comment]
Thank you for the fix, @bryanwweber . The current CI failures (`h5py`?) are unrelated, so I am merging.

**Comment by bryanwweber** [comment]
Yeah it looks like conda forge messed up something with a newer version 🤷‍♂️

**Pull request merged**